frontend/src/components/widgets/ActionLogCards.js

```diff
@@ -35,14 +35,12 @@ const useStyles = makeStyles((theme) => ({
     variant: "outlined",
     color: "default",
     size: "small",
-    
-    ["@media (max-width:620px)"]: {// eslint-disable-line no-useless-computed-key
+    "@media (max-width:620px)":
       fontSize: "0.7rem",
```

Comment:
This line gives us an error when trying to run it. I don't think this works and I think we need to revert back to having the ESLint warnings and deal with it